app/views/layouts/_top_header.html.haml

```diff
@@ -0,0 +1,52 @@
+.service-header
+  .div-nav
+    %nav.navbar.navbar-static-top.navbar-expand-lg.px-5
+      %div
+        = image_tag('coffee.jpg', class: "d-inline-block align-top logo-img")
+        = link_to home_path, id: 'logo-text', class: 'navbar-brand home-page-text' do
+          EUREKA
+          %br/
```

Comment:
don't use br tags. they are not responsive, can't be manipulated from css. just add a div

Reply:
solved. ✓